README.md

```diff
@@ -92,6 +92,7 @@ A curated list of awesome [Apache Spark](https://spark.apache.org/) packages and
 * [KeystoneML](http://keystone-ml.org/) - Type safe machine learning pipelines with RDDs.
 * [JPMML-Spark](https://github.com/jpmml/jpmml-spark) - PMML transformer library for Spark ML.
 * [Distributed Keras](https://github.com/cerndb/dist-keras) - Distributed deep learning framework with PySpark and Keras.
+* [ModelDB](https://mitdbg.github.io/modeldb) - A system to manage machine learning models
```

Comment:
- Missing full stop at the and of the line.
- Since project is not Spark specific I would like to see a bit more precise description.